native/src/hooks/useLoadCities.ts

```diff
@@ -1,24 +1,43 @@
 import { useCallback } from 'react'
 
-import { CityModel, createCitiesEndpoint, ReturnType, useLoadAsync } from 'api-client'
+import { CityModel, createCitiesEndpoint, fromError, ReturnType, useLoadAsync } from 'api-client'
 
+import { SnackbarType } from '../components/SnackbarContainer'
 import dataContainer from '../utils/DefaultDataContainer'
 import { determineApiUrl } from '../utils/helpers'
 
-const useLoadCities = (): ReturnType<CityModel[]> => {
-  const load = useCallback(async () => {
-    if (await dataContainer.citiesAvailable()) {
-      return dataContainer.getCities()
-    }
+type Params = {
+  forceUpdate?: boolean
+  showSnackbar: (snackbar: SnackbarType) => void
+}
+
+const loadWithCache = async ({ showSnackbar, forceUpdate = false }: Params): Promise<CityModel[] | null> => {
+  const cachedData = (await dataContainer.citiesAvailable()) ? await dataContainer.getCities() : null
+
+  if (!forceUpdate && cachedData) {
+    return cachedData
```

Comment:
We never actually invoke this with `forceUpdate=true` anywhere. This means if a user never pulls to refresh in Landing no new content will be loaded? Or does this just work because we invalidate our cache every time we update our app?

Reply:
@sarahsporck however, this leads to the showing of a snackbar on the landing page if offline (even if the user did not manually pull to refresh). Any ideas on how to circumvent this? Or is that even okay behavior?

Reply:
I think this is okay. To circumvent we'd have to throw the error and add the showSnackbar in place instead of passing it. Too much work 🙈

Reply:
Agreed :+1: